PostArticle.jsp

```diff
@@ -136,7 +117,7 @@ footer
 </tr>
 
 <tr>
-<td><input type="text" name="author" value="<c:out value="${ email}"></c:out>" readonly style="background:lightgray"></td>
+<td><input type="text" name="author" value="<c:out value="${ author}"></c:out>" readonly style="background:lightgray"></td>

```

Comment:
Thanks @hema-001 

Why do you always leave a space after `${`?

For example, `${ author}$.`  Why not `${author}$` or `${ author }$`?

**Hui**

Reply:
I'm not sure.
Maybe a habit developed back in the days when I first learned JSP.

Not only me though, even @mjhmohamed is doing the same thing(same school effect i guess).

IBRAHIM